### DISCUSSION TITLE
daisy ui is not working on navbar

### DISCUSSION THREAD

**Discussion by Abdullah0095** [vercel/community#7362] [Help]
### Summary

after uploading navbar is not working. I am sending you two different pictures, one is on my local desktop the other is on a 
![ondesktop](https://github.com/vercel/community/assets/67525899/abfdbce3-f6a0-4d8e-94ab-54a772e83eb6)
![onvercel](https://github.com/vercel/community/assets/67525899/895994bc-bc16-4cdd-be2b-a82057aed2c5)
vertical

### Steps to Reproduce

give me any message how i can overcome the problem

**Comment by pawlean** [comment]
Hi @Abdullah0095!

It's difficult to help you here without looking at your code. But my hunch is that when you deploy your website onto Vercel, it's not picking up the CSS for your navigation bar. I'd suggest looking into [configuring your build](https://vercel.com/docs/deployments/configure-a-build#root-directory) to see if there are any settings here you can adjust (e.g. Root Directory) to fix this issue.

[a comment by an automated account is not shown]
